Packs/SentinelOne/ReleaseNotes/2_1_0.md

```diff
@@ -0,0 +1,12 @@
+##### SentinelOne v2
```

Comment:
Please regenerate the release notes file to match the xsoar standard, using the update-release-notes command